tools/layer-zero-example/hardhat.config.js

```diff
@@ -0,0 +1,226 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Hardhat Example
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+require('dotenv').config();
+require("@nomicfoundation/hardhat-toolbox");
+
+module.exports = {
+  solidity: {
+    compilers: [
+      {
+        version: '0.8.22',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
+          },
+        },
+      },
+    ],
+  },
+  defaultNetwork: 'hedera_testnet',
+  networks: {
+    hedera_testnet: {
+      url: 'https://testnet.hashio.io/api',
+      accounts: [process.env.HEDERA_PK],
+      chainId: 296,
+    },
+    bsc_testnet: {
+      url: "https://data-seed-prebsc-1-s1.binance.org:8545",
+      chainId: 97,
+      gasPrice: 20000000000,
+      accounts: [process.env.BSC_PK]
+    },
+  }
+};
+
+const getEndpointAddress = (network) => {
+  let ENDPOINT_V2;
+
+  // we're using the official LZ endpoints
+  // a list of all endpoint addresses can be found here https://docs.layerzero.network/v2/developers/evm/technical-reference/deployed-contracts
+  if (network === 'hedera_testnet') {
+    ENDPOINT_V2 = '0xbD672D1562Dd32C23B563C989d8140122483631d';
+  } else if (network === 'bsc_testnet') {
+    ENDPOINT_V2 = '0x6EDCE65403992e310A62460808c4b910D972f10f';
+  }
+
+  return ENDPOINT_V2;
+}
+
+task('deploy-whbar', "Deploy WHBAR")
+  .setAction(async (taskArgs, hre) => {
+    const contractFactory = await ethers.getContractFactory('WHBAR');
+    const contract = await contractFactory.deploy();
+    await contract.deployTransaction.wait();
+
+    console.log(`(${hre.network.name}) WHBAR to: ` + contract.address);
```

Comment:
Can you log the tx hashes in these tasks.
Would be nice to follow the transactions in an E2E run